koa-crud/src/data-access/mongoose/vendors/actions.ts

```diff
@@ -1,38 +1,66 @@
-import { VendorsStore } from '../../../types/index';
-import { Vendor as VendorModel } from '../../../lib/mongoose/models/vendor';
+import VendorModelType, {
+  Vendor,
+  VendorDocument,
+} from '../../../lib/mongoose/models/vendor';
 
-const actions = ({ Vendor }: { Vendor: typeof VendorModel }): VendorsStore => {
+type Information = {
+  name?: Vendor['name'];
+  type?: Vendor['type'];
+};
```

Comment:
Can you rename this to `VendorInformation`, to make this distinctive.

Reply:
noted on this